### PR TITLE
Adding functionality for Enquiry viewing and editing

### PR DESCRIPTION
**Pull request by wilwong89**
# Description
Enable navigators to view and edit enquiry information in a form.
Shows related submissions.

For reference: [JIRA PADS-151](https://apps.nrs.gov.bc.ca/int/jira/browse/PADS-151)

## Types of changes

New feature (non-breaking change which adds functionality)

## Checklist

- [X] I have read the [CONTRIBUTING](CONTRIBUTING.md) doc
- [X] I have checked that unit tests pass locally with my changes
- [X] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Review by kyle1morel — CHANGES_REQUESTED** [review]
UI does not match design.
- Info tab should say Information
- Labels are not bold
- Some labels have incorrect capitalization
- Submission date is not a Calendar component
- Relationship to activity is not a LOV
- Contact phone number is not using an input mask
- Section separators are just plain text
- Enquiry detail separator is missing
- Submission state separator is missing
- Waiting on is above the enquiry description text area
- Intake state and Assigned to are in wrong order
- Submit/Cancel buttons are misaligned

Notes tab should probably say `Coming soon` as it's an upcoming feature.

**Comment by wilwong89** [comment]
For reference: [JIRA PADS-151](https://apps.nrs.gov.bc.ca/int/jira/browse/PADS-151)

**Comment by Subin1Doo** [comment]
Hey Wilson, looks like we're missing the "enquiry state" and "waiting on" field under the submission state section. Thanks!

File: frontend/src/store/enquiryStore.ts [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+  // Getters
+  const getters = {
+    // getDocuments: computed(() => state.documents.value),
+    getNotes: computed(() => state.notes.value),
+    // getPermits: computed(() => state.permits.value),
+    getEnquiry: computed(() => state.enquiry.value)
+  };
+
+  // Actions
+  // function addDocument(data: Document) {
```
Comment (sanjaytkbabu) [inline frontend/src/store/enquiryStore.ts]: consider getting rid of comments if not required

**Review by sanjaytkbabu — APPROVED** [review]

File: frontend/src/views/housing/ShasEnquiryView.vue
Comment (kyle1morel) [inline frontend/src/views/housing/ShasEnquiryView.vue:1]: Both the Enquiry intake, and navigator Enquiry form are being used here. There are different concepts and should be in their own views.

**Review by kyle1morel — CHANGES_REQUESTED** [review]

**Comment by slhurley** [comment]
As Subin mentioned, there are a couple of fields missing. Also,
- Submission Type is not being saved when a Nav changes it.  It goes back to blank after any Save.
- There is no toast on Cancel so it is not obvious that anything has been cancelled...nothing moves.
- On the Project Submissions page, if you have an invalid field up higher than the Save button, it isn't clear what happened as the red highlighted field is out of the screen view...no toast or no movement to take you to the problem field.

**Comment by Subin1Doo** [comment]
Based on Sharolyn's comments
**"There is no toast on Cancel so it is not obvious that anything has been cancelled...nothing moves."**
- When a user clicks cancel after making a change -> cancel banner with a message that says "your changes have not been saved."
- When a user makes a change, doesn't save nor cancel, and tries to proceed to another page either by clicking on "back to submissions" or any other menu button, confirmation pop-up message should show up (Basically the same confirmation pop up that shows up in the intake form when they try to leave without saving as draft) "**Leave this page?(head) Are you sure you want to leave this page? Any unsaved changes will be lost." Button - Leave / cancel**

**"On the Project Submissions page, if you have an invalid field up higher than the Save button, it isn't clear what happened as the red highlighted field is out of the screen view...no toast or no movement to take you to the problem field."**
- When they try to save without filing in any required field - can we make it jump to that field, if more than one missing field, jump to whichever one is located up higher.

**Review by kyle1morel — APPROVED** [review]

**Comment by slhurley** [comment]
If an Enquiry is saved as a draft and you go back in to it via the View Drafts and click Submit it doesn't do anything.

**Pull request merged**